### PR TITLE
Homework 19

### PR DESCRIPTION
**Pull request by litserkovna**

### CODE REVIEW

File: src/homework19/PageObjects/MainPage/MainPage.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+export default class MainPage {
+    constructor(page) {
+        this._page = page;
+        this._signupButton = page.locator('.hero-descriptor_btn.btn.btn-primary');
+    }
+    
+    async navigate() {
+        await this._page.goto("");
+    }
```
Comment (sttaran) [inline src/homework19/PageObjects/MainPage/MainPage.js:13]: create BasePage and move it there

File: src/homework19/PageObjects/Registration/RegistrationComponent.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+export default class RegistrationComponent {
```
Comment (sttaran) [inline src/homework19/PageObjects/Registration/RegistrationComponent.js]: RegistrationPopup or RegistrationPopupComponent

File: src/homework19/PageObjects/SettingsPage/SettingsPage.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    constructor(page){
+        this._page = page;
+        this._settingsButton = page.locator('a.btn.btn-white.btn-sidebar.sidebar_btn', { hasText: 'Settings' });
+        this._deleteMyAccountButton = page.locator('button.btn.btn-danger-bg', { hasText: 'Remove my account' });
+        this._confirmDeleteButton = page.locator('button.btn.btn-danger', { hasText: 'Remove' });
+    }
+
+    async openSettings(){
+        await this._settingsButton.click();
+    }
```
Comment (sttaran) [inline src/homework19/PageObjects/SettingsPage/SettingsPage.js]: better use navigate method from BasePage

**Review by sttaran — CHANGES_REQUESTED** [review]

File: src/homework19/PageObjects/MainPage/MainPage.js (hunk 1 above)
Comment (litserkovna) [reply src/homework19/PageObjects/MainPage/MainPage.js:13]: Done ✅

File: src/homework19/PageObjects/Registration/RegistrationComponent.js (hunk 2 above)
Comment (litserkovna) [reply src/homework19/PageObjects/Registration/RegistrationComponent.js]: Updated to RegistrationPopup

File: src/homework19/PageObjects/SettingsPage/SettingsPage.js (hunk 3 above)
Comment (litserkovna) [reply src/homework19/PageObjects/SettingsPage/SettingsPage.js]: ✅

**Pull request merged**